modular-docs-manual/content/topics/module_modular-documentation-repositories.adoc

```diff
@@ -0,0 +1,141 @@
+* `pantheon2.yml`: YAML file with metadata that is required for publishing content on Pantheon 2:
++
+.Example `pantheon2.yml` file
+[source,yaml]
+----
+server: http://localhost:8080
+repository: mod-doc-repo-example
+variants:
+   - name: global-attributes
+     path: _artifacts/document-attributes.adoc
+     canonical: true
+   - name: community-attributes
+     path: _artifacts/document-attributes-kogito-comm.adoc
+   - name: enterprise-attributes
+     path: _artifacts/document-attributes-kogito-ent.adoc
+assemblies:
+  - assemblies/*.adoc
+modules:
+  - modules/*/*.adoc
+resources:
+  - _images/*/*.png
+  - _artifacts/*.adoc
+----
++
+For information about configuring this file for your specific content, contact a member of the CCS tooling team.
+
+This repository structure offers the following potential advantages:
+
+* *Flexibility in determining publishable titles.* In this structure, the images, artifacts, assemblies, and modules all serve as distinct building blocks that you use to compile a document or _title_. You can include an image in multiple modules, a module in multiple assemblies, and an assembly in multiple titles. Additionally, just as an assembly can include multiple modules, a title can contain multiple assemblies.
+* *More intuitive content reuse.* When you decouple the standalone content components from the publishable titles, you avoid potentially awkward includes when modules or assemblies from one title are reused in another. For example, consider a repository where each publishable title folder contains all modules or assemblies or both directly within it. This creates a dependency for that module or assembly content on that title. As you reuse content across title folders, you end up with includes that direct to various other unrelated titles to retrieve the content. Alternatively, in the recommended repository structure, you can reuse all content independent of the titles in which you include it.
+* *Potentially easier collaboration in community and between content repositories.* For repositories that are single sourced with community projects or that need to reuse content between product documentation repositories, this structure enables you to isolate the enterprise title compilation requirements so that the standalone components can be shared more readily. However, single sourcing between community and enterprise content repositories poses other challenges beyond the scope of this document. (See the disadvantages and community considerations sections that follow.)
```

Comment:
s/single sourcing/single-sourcing

Reply:
Yeah, I've also said "single-sourcing", but a growing counter argument is that it's only hyphenated because it's a clipped form of the full phrase "single-sourced publishing/content/whatever". But you're right, definitely becoming the norm and very logical. Changed throughout.